.github/workflows/test.yml

```diff
@@ -1,12 +1,19 @@
 name: Test
 
-on: push
+on:
```

Comment:
@aetheryna Why do we have these changes again? 🤔
Is this because of haven't rebased yet?

See also: https://github.com/aetheryna/google_scraper_ruby/pull/26/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88

Reply:
Oh it's because I merged #26 to this branch so it's showing up here as well 😖 There were no changes done to this file on this branch 😓 (whoops)

Basically these files can be ignored on this branch as it was already reviewed on #26
- `.github/workflows/review_code.yml`
- `.github/workflows/test.yml`
- `.stylelintignore`
- `Dangerfile`

Reply:
@aetheryna So should we pull the latest codebase and rebase with that?
It should work I think 😉

Reply:
@Lahphim oh this is the latest codebase because I merged #26 into this branch :grimacing: